pkg/plugin/v3/scaffolds/internal/templates/config/controller/controller_suitetest.go

```diff
@@ -153,6 +153,7 @@ var _ = BeforeSuite(func(done Done) {
 	By("bootstrapping test environment")
 	testEnv = &envtest.Environment{
 		CRDDirectoryPaths: []string{filepath.Join({{ .CRDDirectoryRelativePath }}, "config", "crd", "bases")},
```

Comment:
```suggestion
		CRDDirectoryPaths:     []string{filepath.Join({{ .CRDDirectoryRelativePath }}, "config", "crd", "bases")},
```

Reply:
Yep, caught that when i saw the generated code. Thanks.